src/pages/reservations/ReservationCard.js

```diff
@@ -0,0 +1,64 @@
+import { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import './ReservationCard.css';
+import { removeReservation } from '../../api/reservationApi/reservation';
+import { clearReservation } from '../../slices/reservationSlice/reservationSlice';
+import { setAlert } from '../../slices/appSlice/appSlice';
+
+const ReservationCard = ({
+  id,
+  startTime,
+  endTime,
+  duration,
+  imageUrl,
+}) => {
+  const dispatch = useDispatch();
+  const [removing, setRemoving] = useState(false);
+  const token = useSelector((state) => state.auth.token);
+
+  const handleClick = () => {
+    setRemoving(true);
+    removeReservation(token, id).then((res) => {
+      if (res.status === 204) {
+        dispatch(clearReservation(id));
+      } else {
+        dispatch(setAlert('Canceling reservation failed!'));
+      }
+      setRemoving(false);
+    });
+  };
+
+  return (
+    <div className="reservation_card">
+      <img src={imageUrl} alt="plane" width="250" height="200" />
+      <p>
+        <strong>Start Time: </strong>
+        {' '}
+        {startTime}
+      </p>
+      <p>
+        <strong>End Time: </strong>
+        {' '}
+        {endTime}
+      </p>
+      <p>
+        <strong>Duration: </strong>
+        {' '}
+        {duration}
+      </p>
```

Comment:
I don't know if you intend to handle this logic on the frontend or back end, anyway. Please make sure that client data are human-readable data:

For example duration: 10 mins(I don't know if it is in mins or not please handle this) instead of duration 10.

Start time, end time should be also humanly readable: meaning, Jun 10 2001 or something anything that you see fit.
![Screenshot from 2023-04-06 21-37-59](https://user-images.githubusercontent.com/105173981/230426917-529edec3-d9f5-40f2-a047-f68bc5f585fc.png)